README.md

```diff
@@ -47,6 +47,13 @@ A template for a secrets file is in the the configuration directory.
 That template shouldn't be modified but should be copied and that copy
 should be read from the separate secure directory.
 
+When running locally the *startup.sh* script assumes that the
+secrets.py file is available in the subdirectory *./OPT/SECRETS*.
```

Comment:
Here and in a code comment below, the name of the directory is written as "`./OPT/SECRETS`".  That is, in uppercase letters and as a relative path (because it begins with "`./`").

However, in the code itself only mentions "`/opt/secrets`", in lowercase letters and as an absolute path.

The documentation and comment should be changed.  Inaccurate documentation is worse than none at all.

Reply:
Which code are you referring to?

Reply:
See: https://github.com/tl-its-umich-edu/kartograafr/pull/15/files#diff-d0486b0d7b181591f52bad014cd3a06fR21